src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/BitcoinActivationRule.cs

```diff
@@ -2,22 +2,19 @@
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
 using NBitcoin;
+using NBitcoin.Networks;
 using Stratis.Bitcoin.Consensus.Rules;
 using Stratis.Bitcoin.Utilities;
 
 namespace Stratis.Bitcoin.Features.Consensus.Rules.CommonRules
 {
     /// <summary>
-    /// Checks if <see cref="Block"/> has a valid PoS header.
+    /// Check that a <see cref="BitcoinMain"/> network block has the correct version the defined active deployments.
```

Comment:
has the correct version the defined active deployments
->
has the correct version according to the defined active deployments

Reply:
Better yes